includes/collection/class-followers.php

```diff
@@ -369,6 +374,13 @@ public static function count_followers( $user_id ) {
 	 * @return array The list of Inboxes
 	 */
 	public static function get_inboxes( $user_id ) {
+		$cache_key = sprintf( self::CACHE_KEY_INBOXES, $user_id );
+		$inboxes = wp_cache_get( $cache_key );
```

Comment:
Instead of using a very verbose cache key, you'd rather want to use the `$group` parameter of the `wp_cache_*` calls.

Reply:
...and the user_id as key?